spec/fabricators/post_fabricator.rb

```diff
@@ -60,6 +60,20 @@
 HTML
 end
 
+Fabricator(:post_with_local_dates, from: :post) do
+  raw <<~RAW
```

Comment:
I do worry this Fabricator is overly specific and not reusable -- for instance I wonder if it was used twice in the same spec, maybe it would error because the content is the same? Also this is a plugin fabricator so it should not be defined in core (for chat we have put the plugin ones in plugins/chat/spec/fabricators/chat_fabricator.rb). Finally, we shouldn't directly apply the `cooked` inside the fabricator since it can get out of date based on markdown rules. Better to just do `post.rebake!` inside the spec, or even better use the `create_post` helper which does that for you https://github.com/discourse/discourse/blob/08eeaa78d055b2e7868e19fa45a39034b4f857fc/spec/support/helpers.rb#L47-L52